lib/appRelationsDiscovery.js

```diff
@@ -0,0 +1,65 @@
+const {validate} = require('./utils');
+
+const supportedSyntax = ['default','mermaid'];
+
+// eslint-disable-next-line sonarjs/cognitive-complexity
+async function getRelations(asyncApiDocs,requestedSyntax = 'default') {
+  if (typeof asyncApiDocs !== 'object') throw new Error('You must pass an array of AsyncApiDocuments on which you wish to discover the relations between');
```

Comment:
When you need as argument an array, you should use the `Array.isArray` function

Reply:
Really sorry for this one

Reply:
No sorry, really, I am happy that I could help :)